### PR TITLE
CCS-3716: Add more logging for golang app

### PR DESCRIPTION
**Pull request by aprajshekhar**
This PR:

1. Logs  server, directory, and user being used for calling uploader in GoApp.
2. Checks whether error is null or not before logging error in GoApp.
3. Logs the response received from Pantheon in uploader so that GoApp can log the actual response text from pantheon rather than reason (OK/Internal Server Error etc.)

### CODE REVIEW

File: tools/git2pantheon/git_uploader.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func gitClone(repository string, branch string, directory string) {
 
 func getUploader() {
 	if _, err := os.Stat("./pantheon.py"); os.IsNotExist(err) {
-		const uploader_url = "https://raw.githubusercontent.com/redhataccess/pantheon/master/uploader/pantheon.py"
+		const uploader_url = "https://raw.githubusercontent.com/aprajshekhar/pantheon/ap_request_data_log/uploader/pantheon.py"
```
Comment (carlosmunoz) [inline tools/git2pantheon/git_uploader.go:36]: Is this change intentional?

Comment (carlosmunoz) [reply tools/git2pantheon/git_uploader.go]: Your latest commit answered my question 😄

Comment (aprajshekhar) [reply tools/git2pantheon/git_uploader.go]: It was unintentional. Thanks for pointing out. Pushed commit fixing it.

**Comment by xdavidson** [comment]
AP, great work on improving the logging. Can you also update the _print_response in createVariant method? It's important see some logging around variant creation.

**Comment by aprajshekhar** [comment]
> AP, great work on improving the logging. Can you also update the _print_response in createVariant method? It's important see some logging around variant creation.

Lisa, thanks. I have updated the _print_response in createVariant method.

**Comment by aprajshekhar** [comment]
Lisa, thanks :)
